src/providers/Project/index.tsx

```diff
@@ -269,7 +269,7 @@ export const ProjectProvider: React.FC<ProjectProviderProps> = ({
         active.index,
       );
 
-      const addr = project.accounts[active.index].address;
+      const addr = project.accounts[accountId].address;
```

Comment:
since we already have this variable on L264
```suggestion
      const addr = deployAccount.address;
```